biojava-structure/src/main/java/org/biojava/nbio/structure/align/ce/CECalculator.java

```diff
@@ -957,9 +955,7 @@ private void checkBestTraces( AFPChain afpChain,
 
 		// removing some loops that are run in orig CE
 		// and which did not do anything
-		if ( debug ){
-			checkPrintRmsdNew(traceMaxSize, winSize, ca1, ca2);
-		}
+		checkPrintRmsdNew(traceMaxSize, winSize, ca1, ca2);
```

Comment:
Here the refactoring is altering what the code actually does: now the checkPrintRmsdNew method will run always (and it performs some calculations that will make things slower). The only way that I see to avoid it is using isDebugEnabled.